server/hwapi/data_models/models.py

```diff
@@ -185,3 +185,9 @@ class Device(Base):
     reports = relationship(
         "Report", secondary=device_report_association, back_populates="devices"
     )
+
+
+class CpuId(Base):
+    __tablename__ = "cpu_id"
+    id_pattern: Mapped[str] = mapped_column(nullable=False, index=True)
```

Comment:
I see in `repository.get_cpu_id_object` that you get all CpuIds then look for the appropriate one in python (not through SQL). So if we're not using the index on `id_pattern` it would be better to remove it